### DISCUSSION TITLE
Fazer uma aplicação

### DISCUSSION THREAD

**Discussion by Nelinha67** [community/community#142863] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Olá

Eu queria saber se alguém está interessado/a em fazer-me uma aplicação para web e pc.

Ver no trade a moeda que está a ser utilizada entre 29 minutos com uma volatilidade elevada.

Informações que dou:
-Date : Friday, 1 November 
-Time : 17:00 PM (GMT)
-Exchange : MEXC.com
-Pairing : USDT o para da direita tipo ???/USDT
eu quero o nome da ????..
e a pesquisa é 30 minutos antes da hora oficial  inicio da procura, o fim será 1 minuto antes da hora oficial.

Queria saber se alguém pode fazer, sou/fui programadora mas já não estou com cabeça para programar.
E qual o valor que me podia ficar.
Penso que é um projecto simples, pois vi alguns projectos por ai referentes a entrar via API ou outro.

Obrigada por lerem

[redacted by moderator]

**Comment by ebndev** [comment]
Hi @Nelinha67, thanks for being a part of the GitHub Community!

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Reply by Nelinha67** [reply]
ok - não sabia
